.travis.yml

```diff
@@ -13,16 +13,21 @@ gemfile:
 install:
   - bundle install --jobs=3 --retry=3 --path=vendor/bundle
 cache:
-  bundler: false
+  bundler: true
   directories:
     - vendor/bundle
+    - vendor/bundle-scaffold
 before_script:
   - unset RACK_ENV
   - unset RAILS_ENV
+  - mkdir -p $PWD/tmp && sudo mount -t tmpfs -o size=1024m tmpfs $PWD/tmp
 matrix:
   allow_failures:
     - rvm: ruby-head
-    - rvm: 2.4.0
+  exclude:
+    - rvm: 2.4.1
```

Comment:
We're excluding just Rails 3 + Ruby 2.4.1 (which are incompatible), not all of 2.4.1.